crates/compiler/parse/src/expr.rs

```diff
@@ -3439,7 +3483,7 @@ fn pat_ends_with_spaces_conservative(pat: &Pattern<'_>) -> bool {
         | Pattern::ListRest(_)
         | Pattern::As(_, _)
         | Pattern::OpaqueRef(_) => false,
-        Pattern::Apply(_, args) => args
+        Pattern::Apply(_, args, _) => args
```

Comment:
This shouldn't be ignoring the PatternApplyStyle. In fact if we're using `ParensAndCommas`, we should just immediately return false here without inspecting the last arg.